bundler/lib/bundler/setup.rb

```diff
@@ -5,6 +5,10 @@
 if Bundler::SharedHelpers.in_bundle?
   require_relative "../bundler"
 
+  if Bundler.settings[:auto_install]
+    Bundler.auto_install
+  end
```

Comment:
Was chatting with @ngan and he suggested this should go into `Bundler.setup`. That would mean you get that anytime you use `Bundler.setup`, not just `require "bundler/setup"`

Does that seem reasonable @deivid-rodriguez ?

Reply:
That seems fine to me, yes.